### PR TITLE
[W5.4][W14-A1]Hou Denghao

### PR DESCRIPTION
**Pull request by HouDenghao**
Trying to fix my mistake

### CODE REVIEW

**Comment by wenzongteo** [comment]
Can consider sorting by other columns of the addressbook!

**Comment by conantteo** [comment]
Good incremental updates on your changes!

File: src/seedu/addressbook/parser/Parser.java [hunk 1]
```diff
@@ -11,17 +11,7 @@
...
-import seedu.addressbook.commands.Command;
-import seedu.addressbook.commands.DeleteCommand;
-import seedu.addressbook.commands.ExitCommand;
-import seedu.addressbook.commands.FindCommand;
-import seedu.addressbook.commands.HelpCommand;
-import seedu.addressbook.commands.IncorrectCommand;
-import seedu.addressbook.commands.ListCommand;
-import seedu.addressbook.commands.ViewAllCommand;
-import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.commands.*;
```
Comment (okkhoy) [inline src/seedu/addressbook/parser/Parser.java]: Don't import using `*` import the classes explicitly, only what is necessary

**Comment by okkhoy** [comment]
@HouDenghao 
Some comments added. Please ack & close the PR after reading comments.

**Comment by HouDenghao** [comment]
I have added my Junit test and next time I will pay attention to the import (do not use *);

**Comment by HouDenghao** [comment]
@okkhoy Hello！How to judge whether my submission meet the requirements ? Do I need to create a new PR as the announcement said?

**Comment by HouDenghao** [comment]
@okkhoy I have done the enhancement to address book-level3 and sent PR. Because I think I did not meet the requirement this time (such as the missing Junit test), I followed the announcement for getting my marks back. If there are still some mistakes, I will correct it as soon as possible before week 6 tutorial. Thank you!

**Pull request closed without merging**